content/influxdb/v2.0/reference/flux/stdlib/slack/endpoint.md

```diff
@@ -54,18 +54,12 @@ _**Data type:** String_
 The output function requires a `mapFn` parameter.
 
 ### mapFn
```

Comment:
Are these mapFn updates applicable for the `http.endpoint`?

Reply:
Yes, updated that as well.